### PR TITLE
Add tooltips for nav/tab sections

### LINKED ISSUE #622: Usability - tooltips
Related to issue #621 which may be confusing for new/novice users, I noticed there are no tooltips on any buttons. Whilst the iconography is fairly standard, when the "new tab" button moves, it is not immediately obvious that the "+" icon still represents "new tab" as its context has changed. Tooltips on hover would assist.

### PR DESCRIPTION
**Pull request by psimyn**
Resolves #622 

Probably needs localisation though

### CODE REVIEW

File: js/components/main.js [hunk 1]
```diff
@@ -224,11 +224,11 @@ class Main extends ImmutableComponent {
       <div className='top'>
         <div className='navigator-wrapper'>
           <div className='backforward'>
-            <span
+            <span title='Go back'
```
Comment (bbondy) [inline js/components/main.js]: These should be defined in the l20n file please, thanks!

Comment (bbondy) [inline js/components/main.js]: `app/locales/en-US/app.l20n`
Example:
`<backButton
  title: "Go back">`

Then set the data-l10n-id

Comment (psimyn) [inline js/components/main.js]: you are a wizard. Was wondering how to set other values. Thanks!

Comment (psimyn) [inline js/components/main.js]: @bbondy updated with data-l10n-ids 

let me know if there is some kind of system in the locale file that I've missed, I just put them near related items

**Comment by bbondy** [comment]
Looks great, pls rebase and then I'll merge in.

**Comment by psimyn** [comment]
rebased. Thanks, @bbondy

**Comment by bbondy** [comment]
Thanks!

**Pull request merged**